code/game/objects/items/devices/handheld_defib.dm

```diff
@@ -63,7 +63,9 @@
 				else
 					to_chat(user, "<span class='danger'>[H] doesn't respond!</span>")
 
-			H.AdjustParalysis(3)
+			if((H.health <= HEALTH_THRESHOLD_CRIT) || (H.undergoing_cardiac_arrest())) //Don't sleep if they aren't in crit
```

Comment:
undergoing_cardiac_arrest won't work seeing it's set to false prior on line 55

Reply:
Moved it and the surrounding stun to above the line that sets heart attack to false